### PR TITLE
Ewl 807 toggle switch component

### PR DESCRIPTION
**Pull request by ra-phael**
https://eurosportdigital.atlassian.net/browse/EWL-807

### CODE REVIEW

File: src/elements/Toggle/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    background-color: ${azureRadiance};
+    transition: 0.4s;
+    border-radius: 50%;
+    ${props => (props.isSetToLeft ? '' : 'transform: translateX(18px);')}
+  }
+`;
+
+const Toggle = ({ toggleCallback, isSetToLeft, leftLabel, rightLabel }) => (
+  <StyledToggleWrapper>
+    <StyledLabel isActive={isSetToLeft} style={{ textAlign: 'right' }}>
```
Comment (val3ntin) [inline src/elements/Toggle/index.js]: just curious, but why don't move the inline `style={....}` directly to the `StyledLabel` component .e.g. by adding `isLeft` boolean prop ?

`<StyledLabel .... isLeft />`

Comment (ra-phael) [reply src/elements/Toggle/index.js]: yes, I considered doing that as well... I think the amount of code and complexity added is about the same. Unless we 100% forbid inline style, here it seems to be a good use case for it.

**Comment by ra-phael** [comment]
Thanks for your feedback @val3ntin ! I just got words that actually we don't want any transition.

File: src/elements/Toggle/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import { coreLightMinus1, azureRadiance } from '../../colors';
```
Comment (JulienKode) [inline src/elements/Toggle/index.js]: Why we do not use a theme for the colors ? 

inside the style we can do something like:
```
const StyledStuff = styled.div`
    background-color: ${( { theme } ) => theme.myFavoriteColor.pink};
 `;
```

We will never want a white theme ?

**Pull request merged**